controller/backup_controller.go

```diff
@@ -207,6 +207,19 @@ func getLoggerForBackup(logger logrus.FieldLogger, backup *longhorn.Backup) *log
 	)
 }
 
+func (bc *BackupController) isBackupBeingUsedForVolumeRestore(backupName, backupVolumeName string) error {
```

Comment:
Suggest `func (bc *BackupController) isBackupBeingUsedForVolumeRestore(backupName, backupVolumeName string) (bool, error)
` instead, because this is used to determine if the backup is used or not.

Reply:
Addressed. Thank you.